.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+## Summary
+
+Please describe your changes and what you plan to introduce as part of this pull request submissions. If it fixes a bug or resolves a feature request, be sure to link to the issue numbers following the template below.
+
+## Pull Request Checklist
+🚨Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.
+
+- [ ] Please ensure that you are making a pull request against the **Development** branch
+- [ ] Please use the `WIP` keyword in the title of your PR if you are not ready for review
```

Comment:
WIP flag was to engage and get early feedback for a PR but the changes are not ready to be merged. Can we change the wording on this?

Reply:
How about:

- Please use the `WIP` keyword in the title of your PR to engage and get early feedback but the changes within the PR is not ready to be merged?

Reply:
Yes, this is perfect.